endpoints/btc/tests/integration_test.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-bitwise */
+/* eslint-disable import/first */
+/* eslint-disable import/extensions */
+const request = require('request')
+const helpers = require('../../../lib/test_helpers.js')
+
+describe('btc', function () {
+  this.timeout(20000)
```

Comment:
@pmm1 is this timeout needed?

Reply:
I copied this from the /aur endpoint, definitely not needed.